### PR TITLE
Fibaro initial support

### PR DESCRIPTION
**Pull request by pbalogh77**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17891

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/binary_sensor.fibaro.markdown [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+description: "Instructions on how to integrate Fibaro binary sensors into Home Assistant."
+date: 2018-11-14 23:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: fibaro.png
+ha_category: Binary Sensor
+ha_iot_class: "Local Push"
+ha_release: 0.82
```
Comment (MartinHjelmare) [inline source/_components/binary_sensor.fibaro.markdown]: 0.83

File: source/_components/fibaro.markdown [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+title: "Fibaro"
+description: "Instructions on how to setup Fibaro Z-Wave hubs (HCL and HC2) and configure devices within Home Assistant."
+date: 2018-11-14 20:04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: fibaro.png
+ha_category: Hub
+ha_release: 0.82
```
Comment (MartinHjelmare) [inline source/_components/fibaro.markdown]: 0.83

File: source/_components/fibaro.markdown [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+---
```
Comment (MartinHjelmare) [inline source/_components/fibaro.markdown:14]: Please add `ha_iot_class`.

File: source/_components/fibaro.markdown [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+The [Fibaro](http://fibaro.com) hub is a controller mainly connecting to Z-Wave devices.
+
+Switches, Lights (inc Dimmers), Locks, Sensors, Binary sensors, Coverss are supported - and will be automatically added when HA connects to your Fibaro controller.
```
Comment (MartinHjelmare) [inline source/_components/fibaro.markdown]: Switches, Lights (including Dimmers), ... Covers are supported - ...

File: source/_components/fibaro.markdown [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  username: your_username
+  password: your_password
+```
+
+{% configuration %}
+url:
+  description: The URL for your Fibaro device.
+  required: true
+  type: string
+{% endconfiguration %}
```
Comment (MartinHjelmare) [inline source/_components/fibaro.markdown:48]: Add all the items in the config spec, ie password, username, plugins.

File: source/_components/fibaro.markdown [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+<p class='note'>
+  It is recommended to assign a static IP address to your Fibaro Controller. This ensures that it won't change IP addresses, so you won't have to change the `url` if it reboots and comes up with a different IP address. See your router's manual for details on how to set this up. If you need the MAC address of your Fibaro, check the label on the bottom.
```
Comment (MartinHjelmare) [inline source/_components/fibaro.markdown]: ... so you won't have to change the `url` if the controller reboots...

File: source/_components/fibaro.markdown [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+</p>
+
+### {% linkable_title Configure devices %}
```
Comment (MartinHjelmare) [inline source/_components/fibaro.markdown]: There's no text under this header?

File: source/_components/fibaro.markdown [hunk 8]
```diff
@@ -25,22 +26,32 @@ fibaro:
   url: http://192.168.1.161/api/
   username: your_username
   password: your_password
+  plugins: true or false (default false)
```
Comment (MartinHjelmare) [inline source/_components/fibaro.markdown]: The first config example should be minimal, with only required items. We can remove this option here.

Comment (frenck) [reply source/_components/fibaro.markdown]: Removed.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
✅ Thanks for the PR, @pbalogh77!

Can be merged as soon as the build passes.

**Pull request merged**